backend/app/assets/javascripts/spree/backend/taxonomy.js.coffee

```diff
@@ -1,9 +1,11 @@
 handle_ajax_error = (XMLHttpRequest, textStatus, errorThrown) ->
-  $.jstree.rollback(last_rollback)
-  $("#ajax_error").show().html("<strong>" + server_error + "</strong><br />" + taxonomy_tree_error)
+  errors = JSON.parse(XMLHttpRequest.responseText).errors
+  errorString = $.map(errors, (value, key) -> (key.charAt(0).toUpperCase() + key.substr(1)) + " " + value).join('\n')
+  $.jstree.rollback(window.last_rollback)
```

Comment:
This causes a category loading forever bug if you call it when there's a "load children of taxon" AJAX  request running